Rubberduck.CodeAnalysis/Inspections/Abstract/IsMissingInspectionBase.cs

```diff
@@ -20,7 +20,7 @@ protected IsMissingInspectionBase(RubberduckParserState state)
         private static readonly List<string> IsMissingQualifiedNames = new List<string>
         {
             "VBE7.DLL;VBA.Information.IsMissing",
-            // TODO find QN for VB6
+            "VBA6.DLL;VBA.Information.IsMissing"
```

Comment:
This feels just teensy bit over-specific. Do we really care whether it's `VBA6` or `VBA7`? If they loaded `VBA5` (woah, stone age!) or one day load `VBA8` then what? FWIW, I think the DLL here is the same between VB6 and VBA; the only difference is that VB6 will be forever using VBA6 (or lower?) for obvious reasons but we don't know for sure whether VBA will forever more be VBA7.

Reply:
This will likely cover 99.99% of use cases. I'm not entirely sure that the VBA namespace can't be hidden by another library (guessing it can't), but I'd rather take the minimal chance that the user has another VBA dll loaded.